app/controllers/targets_controller.rb

```diff
@@ -24,6 +24,20 @@ def create
     end
   end
 
+  def edit
+    target
```

Comment:
You don't need to do this. The helper method variable will initialize the first time you call it.